CMakeLists.txt

```diff
@@ -163,5 +166,22 @@ if(SOCI_TESTS)
   add_subdirectory(tests)
 endif()
 
+###############################################################################
+# build config file
+###############################################################################
```

Comment:
I started to write has you said but I feel that someday it will be unsynchronised. That's why I switch to this "auto" mode.

But this is a minor risk, and this is easy to switch. So @mloskot what do you prefer?